### PR TITLE
apitrace: 7.1.old -> 11.1

### PR DESCRIPTION
**Pull request by K900**
###### Description of changes

Update from 2015 to current latest release. Run-tested on glxgears, though I'd be very surprised if anyone noticed it break anyway.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mweinelt** [comment]
@ofborg build apitrace

File: pkgs/applications/graphics/apitrace/default.nix [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+  libX11,
+  procps,
+  python3,
+  libdwarf,
+  qtbase,
+  wrapQtAppsHook,
+  libglvnd,
+  gtest,
+  brotli,
+}:
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/apitrace/default.nix:16]: Can we please stop using incompatible formatters? Thanks!

File: pkgs/applications/graphics/apitrace/default.nix [hunk 2]
```diff
@@ -1,27 +1,37 @@
...
 
   src = fetchFromGitHub {
-    sha256 = "11bwb0l8cr1bf9bj1s6cbmi77d5fy4qrphj9cgmcd8jpa862anp5";
-    rev = "26966134f15d28f6b4a9a0a560017b3ba36d60bf";
-    repo = "apitrace";
     owner = "apitrace";
+    repo = "apitrace";
+    rev = version;
+    hash = "sha256-rvC6iVWNNxH11hzQvRTo+SQi9jEUCPWGSdJmKJe9SQ0=";
+    fetchSubmodules = true;
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/apitrace/default.nix:27]: We should at least use our zlib, libpng and brotli.

Comment (K900) [reply pkgs/applications/graphics/apitrace/default.nix:27]: I believe it's using those to link statically so we can't really avoid that (unless we do some bad pkgsStatic things)

Comment (SuperSandro2000) [reply pkgs/applications/graphics/apitrace/default.nix:27]: Maybe that would be better than trusting upstream to update those dependencies and fix CVEs?

**Pull request merged**